src/transaction/flashback_cl.c

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * flashback_cl.c -
+ */
+
+#ident "$Id$"
+
+#include "config.h"
+
+#include <stdio.h>
+#if defined(WINDOWS)
+#include <windows.h>
+#else /* WINDOWS */
+#include <sys/ioctl.h>
+#include <unistd.h>
+#endif /* LINUX */
+
+#include "flashback_cl.h"
+#include "object_representation.h"
+
+#if defined(WINDOWS)
+static int
+flashback_util_get_winsize ()
```

Comment:
We need to test on Windows for this. Please add test scenarios for it to the test list.
- If the size of the windows is changed during the execution.
- If the size of the windows is too small to output even one line (need to consider the heading, information messages, etc).